hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/HttpAttributeEnricher.java

```diff
@@ -37,20 +38,20 @@
 
   @Override
   public void enrichEvent(StructuredTrace trace, Event event) {
-    String url = EnrichedSpanUtils.getFullHttpUrl(event).orElse(null);
-    if (url != null) {
-      URL fullUrl = null;
+    String urlString = EnrichedSpanUtils.getFullHttpUrl(event).orElse(null);
```

Comment:
@GurtejSohi Isn't `EnrichedSpanUtils.getFullHttpUrl()` return full URL? It should, otherwise it should return null. Otherwise it shouldn't be named that way :)

Reply:
No, it just returns the url field:
`  public static Optional<String> getFullHttpUrl(Event event) {
    if (event.getHttp() != null && event.getHttp().getRequest() != null) {
      return Optional.ofNullable(event.getHttp().getRequest().getUrl());
    }
    return Optional.empty();
  }`

Yes, I think it should be renamed to `getHttpUrl`.